### DISCUSSION TITLE
Support For Private Repositories using api token?

### DISCUSSION THREAD

**Discussion by FinSmith198** [dvershinin/lastversion#142] [Ideas]
I do not believe this is already a feature, as it did not work for me, but it would be great if i could fetch the latest release version from my private repositories.

**Comment by dvershinin** [comment]
It works for me (c). 

So please check if your token is correct and belongs to the right account.

**Reply by FinSmith198** [reply]
Just to check, This uses the Classic personal access tokens over on https://github.com/settings/tokens/new, and not those new fine grained tokens which i don't know much about

**Reply by dvershinin** [reply]
I think both should work, I tested with classic though.

**Reply by FinSmith198** [reply]
I have found my problem, sorry for pestering a little, i'm quite new to linux, ![image](https://github.com/dvershinin/lastversion/assets/137229198/d2d344c3-9835-40ba-ba97-7c81ba8b9683)

I in fact forgot to run the source command properly after adding my token to .bashrc, so it didn't set the env variable correctly.
Thanks for the help!
